jekyll.gemspec

```diff
@@ -29,7 +29,7 @@ Gem::Specification.new do |s|
   s.rdoc_options     = ["--charset=UTF-8"]
   s.extra_rdoc_files = %w(README.markdown LICENSE)
 
-  s.required_ruby_version     = ">= 2.5.0"
+  s.required_ruby_version     = ">= 2.7.0"
```

Comment:
If you roll this back to 2.5.0, does that solve the JRuby failure? Or are we using syntax features from CRuby that it doesn't support?

Reply:
Reverted changes..